src/main/java/me/ShermansWorld/AlathraExtras/woodcutting/Woodcutting.java

```diff
@@ -0,0 +1,79 @@
+package me.ShermansWorld.AlathraExtras.woodcutting;
+
+import me.ShermansWorld.AlathraExtras.AlathraExtras;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.Tag;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.StonecuttingRecipe;
+import org.bukkit.plugin.Plugin;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class Woodcutting {
+
+    public static void setWoodcuttingRecipes() {
+        String[] woodcuttingItems = {
+            "BUTTON",
+            "FENCE",
+            "FENCE_GATE",
+            "PLANKS",
+            "PRESSURE_PLATE",
+            "SIGN",
+            "SLAB",
+            "STAIRS",
+            "TRAPDOOR",
+            "DOOR"
+        };
+
+        int[] woodcuttingCount = {
+            4,
+            8,
+            4,
+            4,
+            4,
+            4,
+            8,
+            4,
+            4,
+            1
+        };
```

Comment:
```suggestion		
        Map<String, Integer> map = Map.of(
			"BUTTON", 4,
			"FENCE", 8,
			"FENCE_GATE", 4,
			"PLANKS", 4,
			"PRESSURE_PLATE", 4,
			"SIGN", 4,
			"SLAB", 8,
			"STAIRS", 4,
			"TRAPDOOR", 4,
			"DOOR", 1
		);
```
This is less verbose and error-prone if someone were to modify this in the future.